.github/workflows/upload_component.yml

```
@@ -0,0 +1,20 @@
+name: Push components to Espressif Component Service
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  upload_components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Upload components to component service
+        uses: espressif/github-actions/upload_components@master
+        with:
+          directories: "components/esp_modem"
```

In the current implementation of upload components action, 
if `directories` have only 1 item, then `name` is mandatory.

```suggestion
          directories: "components/esp_modem"
          name: "esp_modem"
```

---

Thank you, applied!